### PR TITLE
TIC-104: Remove unused files

### PR DESCRIPTION
**Pull request by HaliTran**
### Description
Removed unused files within the repo. Did my best to look through the codebase but there could be more. There seems to be a lot of dependencies, even in files that are no longer used.

### Risks
Most of the deletion occurred within the old Events directory so this may affect how the events page function, but from my testing, nothing was affected.

### Validation
Vigorous testing of the removed files (especially events) to ensure it functions as intended

### Issue
https://pdx-hkaus.atlassian.net/jira/software/projects/TIC/boards/1?selectedIssue=TIC-104

### Operating System
M2 Macbook

### CODE REVIEW

**Review by jonathantnguyen — DISMISSED** [review]
Manual regression testing shows no adverse affect on the build. Would like others to confirm.

Win10/Chrome

File: client/src/components/Ticketing/ticketingmanager/Events/deleteConfirm_depreciated.tsx [hunk 1]
```diff
@@ -4,27 +4,27 @@ import React, {CSSProperties} from 'react';
  * @property {object} popUpContainer - A property container for pop up
  */
 const popUpContainer: CSSProperties = {
-  'position': 'fixed',
-  'background': '#00000050',
-  'width': '100%',
-  'height': '100vh',
-  'top': '0',
-  'left': '0',
+  position: 'fixed',
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Events/deleteConfirm_depreciated.tsx:7]: Any reason why this file wasn’t deleted as well?

File: client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer_deprecated.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ import DeleteConfirm from './deleteConfirm_depreciated';
 export interface MapPropsToShowingInputContainer {
   showingData: Showing;
   eventinstanceid: number;
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer_deprecated.tsx:28]: And this file too

File: client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer_deprecated.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ import DeleteConfirm from './deleteConfirm_depreciated';
...
-  index: number,
+  index: number;
   handleSetShow: (show: Showing) => void;
   handleDeleteShow: (e: any) => void;
 }
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer_deprecated.tsx:32]: You can probably remove this file if you find a good spot to put the `toDateStringFormat()` function

File: client/src/components/Ticketing/ticketingmanager/Events/deleteConfirm_depreciated.tsx (hunk 1 above)
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Events/deleteConfirm_depreciated.tsx:7]: Hmm when I removed it initially, there were dependency errors but now when I check, it seems to have been resolved. Not sure why, but I'll remove it

File: client/.eslintrc.json [hunk 4]
```diff
@@ -26,7 +26,7 @@
     "no-unused-vars": "off",
     "valid-jsdoc": "off",
     "operator-linebreak": "off",
-    "indent": ["warn", 2],
+    "indent": "off",
```
Comment (henrykaus) [inline client/.eslintrc.json:29]: probably for the best

**Review by henrykaus — APPROVED** [review]

**Pull request merged**